core/request.go

```diff
@@ -101,28 +97,18 @@ func makeRequestHandler(id, n uint32, view viewProvider, verifier messageSignatu
 			return false, err
 		}
 
+		if new = seqAcceptor(request); !new {
+			return false, nil
+		}
+
 		view := view()
 		primary := isPrimary(view, id, n)
-		// The primary will now generate a Prapare message for
-		// this request. In that case, the consensus protocol
-		// is started for this request and the request
-		// identifier should be accepted.
-		new, err = seqAcceptor(request, prepared || primary)
-		if err != nil {
-			err = fmt.Errorf("Failed to check/accept request ID: %s", err)
-			return false, err
-		}
 
-		// TODO: The request timer should be started in backup
-		// replicas when received a new request from a client.
-		// It must probably be done atomically with checking
-		// the request ID, otherwise there might be a race
-		// condition between checking the request ID and
-		// starting the timer here versus stopping the timer
-		// and supplying a Reply message when the request gets
-		// executed.
+		// TODO: A new request ID has arrived; the request
+		// timer should be re-/started in backup replicas at
+		// this point.
```

Comment:
Does this timer mean the timer for a view change described in page 6 in [Efficient Byzantine Fault-Tolerance paper](https://www.researchgate.net/publication/260585535_Efficient_Byzantine_Fault-Tolerance) and #5?